client/js/components/SunburstSlice.js

```diff
@@ -50,13 +50,15 @@ export default class SunburstSlice extends React.Component{
         const childrenArcDescs = pie(children.map(c => c.total));
         const parentArcDesc = { startAngle, endAngle };
 
-        const selected = highlightedNodes && highlightedNodes.has(node);
+        const highlighted = highlightedNodes && highlightedNodes.has(node);
+        const selected = selectedNode === node;
```

Comment:
Petite préférence pour utiliser des préfixes comme `is` ou `has` pour signaler des états – `isSelected` pour spécifier si `selectedNode` etc.